src/applications/ivc-champva/shared/tests/helpers.js

```diff
@@ -162,13 +162,16 @@ export function verifyAllDataWasSubmitted(data, req) {
   describe('Data submitted after E2E runthrough should include all data supplied in test file', () => {
     Object.keys(data).forEach(k => {
       // handle special cases:
-      if (k.includes('DOB') || k.includes('Date')) {
+      if (k.includes('DOB') || k.includes('Date') || k === 'missingUploads') {
```

Comment:
`missingUploads` is an array without a set order, so E2E test was flaky if the order was different after submitting. Fixed by just checking that length is where we want it since the inner details aren't relevant for this prop.